src/XPurchase/VirtoCommerce.XPurchase/Schemas/CartGiftItemType.cs

```diff
@@ -0,0 +1,14 @@
+using GraphQL.DataLoader;
+using MediatR;
+using VirtoCommerce.ExperienceApiModule.Core.Services;
+
+namespace VirtoCommerce.XPurchase.Schemas
+{
+    public class CartGiftItemType : LineItemType

```

Comment:
This type is seemed  redundant I think we should use the single type GiftItemType for available and accepted gifts representation